challengeTwo/usersprompt.js

```diff
@@ -0,0 +1,26 @@
+var readline = require('readline');
+
+var rl = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout
+});
+
+function enterYear(number) {
+    if(number<18){
+console.log('User is a child');
+} 
+else if(number<36){
+console.log('User is a youth')
+} 
+else{
+console.log('User is an adult')
+}  
+}
+
+rl.question('What is your Year of Birth?: ', function (x) {
+         var sum = 2019 - parseFloat(x);

```

Comment:
Why are you parsing the value you get into a float value? use `parseInt` instead since we ar dealing with integers in this case

Altrnatively,
JavaScript gives you other ways of casting string to integers which include:
```js
var myVal = "12"

// using the number contructor
var myValToNumber = Number(myVal)

// or using the + operator
var myValToNumber = +myVal
```

Reply:
@dj made the corrections

Reply:
@dennisja